tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -354,9 +354,15 @@ private static async Task WriteDiagnosticLog(TracerManager instance)
                     writer.WritePropertyName("routetemplate_expansion_enabled");
                     writer.WriteValue(instanceSettings.ExpandRouteTemplatesEnabled);
 
+                    writer.WritePropertyName("querystring_reporting_enabled");
+                    writer.WriteValue(instanceSettings.QueryStringReportingEnabled);
+
                     writer.WritePropertyName("obfuscation_querystring_regex_timout");
                     writer.WriteValue(instanceSettings.ObfuscationQueryStringRegexTimeout);
 
+                    writer.WritePropertyName("obfuscation_querystring_regex");
+                    writer.WriteValue(instanceSettings.ObfuscationQueryStringRegex.Substring(0, Math.Min(instanceSettings.ObfuscationQueryStringRegex.Length, 200)));
```

Comment:
Should we be truncating the regex in the log? It looks like the length of the default value is 631

Reply:
mhh.. I dont know, I thought it would clogged this log in particular and would make it hard to read🤔  But I can put it all^^

Reply:
> I thought it would clog this log in particular and would make it hard to read🤔

I think that's definitely true 😅  but if we _don't_ include the whole thing, then it's probably not worth writing it at all!

Just a thought - is it possible to only write this key _if_ they're not using the default? That way it won't clog the log for the "normal" case, but we could have the full thing in cases where they're customising it?

Not a blocker, we can easily follow up later if needs be!

Reply:
> Just a thought - is it possible to only write this key if they're not using the default? That way it won't clog the log for the "normal" case, but we could have the full thing in cases where they're customising it?

changed in ef41b066798dd80d0c086f7f766c9a197130d852